### ISSUE TITLE
Can create a fake acct with usc email - Keyboard UI Problems for Register screen ios

### ISSUE THREAD

**Issue by cwmiles** [SCCapstone/CampusConnect#243]
https://user-images.githubusercontent.com/70288320/231045656-f4ab3325-eab6-4f42-818a-73c18b1eb8ad.MOV

Note: Cant screenshot some keyboard issues.

- Not sure if emails are supposed to be verified in some way but potentially could help prevent fakes.

- The keyboard UI problem is when typing in your confirmed password the user can't see the text box or any characters they are inputing because the keyboard is above the input box. When trying to screenshot this, the entire password field is automatically empty in the screenshot and the keyboard isn't shown so you will have to test directly to see.

- Also, the user can hit return to close the keyboard on the register screen, but when entering details about bio, name, year, and major, the return button doesn't close the keyboard.(see video)

**Comment by econnerty** [comment]
Bugs like this should probably be separated.

In regards to the email verification: we tried, but the school automatically filters and clicks on all firebase links for scanning, which would activate the account by default.

The workaround we created ended up being too convoluted, so email verification was disabled.

The school would have to start allowing our emails to pass through the filter, but that is unlikely to happen.

The keyboard thing should be noted as an iOS visual big and be a separate issue.
